.gitignore

```diff
@@ -2,3 +2,4 @@
 /scaler
 /amGithubNotifier
 /commentMonitor
+build/
```

Comment:
why do we need this?

I usually prefer to put most thing in .git/exclude/info locally instead of maintaining a global gitignore file. Most ignores are local env specific so best to keep them as local configs.

Reply:
Will remove it.